src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    FigureSupplier() {
+        random = new Random();
+        colorSupplier = new ColorSupplier();
+    }
```

Comment:
Initialize `Random` and `ColorSupplier` directly in field variables or **with constructor with params.**